functions/lib/cla.js

```diff
@@ -1,6 +1,72 @@
 module.exports = Cla
 
+/**
+ * Returns CLA-related functions.
+ * @param db {FirebaseFirestore.Firestore}
+ */
 function Cla (db) {
+  function getSet (whitelist, typ) {
+    if (!Object.prototype.hasOwnProperty.call(whitelist, typ)) {
+      whitelist[typ] = new Set()
+    }
+    return whitelist[typ]
+  }
+
+  function toJson (whitelist) {
```

Comment:
Do we want to create a Whitelist model in the `common/model` folder?
If so this method should be in there

Reply:
Right now the whitelist collection is used by the backend only, so it's not really common. Do you see any value in consuming it from the UI? If so, then we can create a model class.

Reply:
No not really, was just wondering but I believe it's fine for now

Reply:
Please check the new changes to the whitelist logic. I believe we don't need a model for now.